unit/java_bytecode/java_string_library_preprocess/convert_exprt_to_string_exprt.cpp

```diff
@@ -38,22 +38,35 @@ TEST_CASE("Convert exprt to string exprt")
   preprocess.add_string_type("java.lang.String", symbol_table);
   symbol_typet java_string_type("java::java.lang.String");
   symbol_exprt expr("a", pointer_type(java_string_type));
-  convert_exprt_to_string_exprt_unit_test(
+  refined_string_exprt string_expr=convert_exprt_to_string_exprt_unit_test(
```

Comment:
Again consider structuring this test in a way to highlight what is actually being tested vs. what is setup code